src/ProjectTemplates/Web.Spa.ProjectTemplates/content/Angular-CSharp/appsettings.json

```diff
@@ -4,15 +4,15 @@
 ////#if (UseLocalDB)
 //    "DefaultConnection": "Server=(localdb)\\mssqllocaldb;Database=aspnet-Company.WebApplication1-53bc9b9d-9d6a-45d4-8429-2a2761773502;Trusted_Connection=True;MultipleActiveResultSets=true"
 ////#else
-//    "DefaultConnection": "DataSource=app.db"
+//    "DefaultConnection": "DataSource=app.db;Cache=Shared"
 ////#endif
 //  },
 ////#endif
   "Logging": {
       "LogLevel": {
-      "Default": "Information",
-      "Microsoft": "Warning",
-      "Microsoft.Hosting.Lifetime": "Information"
+      "Default": "Information",
+      "Microsoft": "Warning",
+      "Microsoft.Hosting.Lifetime": "Information"
```

Comment:
⁉ Did this file have mixed Unix and Windows line endings?

Reply:
Definitely not intentional, if that was the case.